### DISCUSSION TITLE
How to set generic Models/loras location

### DISCUSSION THREAD

**Discussion by VeteranXT** [lshqqytiger/stable-diffusion-webui-directml#517] [General]
Title says it all?

**Comment by CS1o** [comment]
Edit your webui-user.bat
At the line set COMMANDLINE_ARGS=
add: --models-dir "D:\Path\to\ModelsFolder"
That uses the same structure as Auto111 so you need a Models/Lora and Models/Stable-Diffusion Folder
As Alternative you can also just use --ckpt-dir (for models) and --lora-dir (for loras).
Same goes for embeddings, controlnets, etc
